power_grid_model_c/power_grid_model/include/power_grid_model/common/exception.hpp

```diff
@@ -57,7 +57,7 @@ class InvalidBranch3 : public PowerGridError {
 class InvalidTransformerClock : public PowerGridError {
   public:
     InvalidTransformerClock(ID id, IntS clock) {
-        append_msg("Invalid clock for transformer " + std::to_string(id) + ", clock  " + std::to_string(clock) + '\n');
+        append_msg("Invalid clock for transformer " + std::to_string(id) + ", clock " + std::to_string(clock) + '\n');
```

Comment:
could this be breaking for our users?

Reply:
We do not guarantee the error messages. So any change in error message as far as I concern no breaking changes.

Reply:
ok great. in that case, do you think it's a good idea to have a look at the other error messages as well? e.g. we use `\n ` with the whitespace after the newline for all errors. Is that intentional?

Reply:
let's revisit later after we've given our users enough time to migrate to the new errors